cni/network/network.go

```diff
@@ -322,6 +323,63 @@
 	log.Printf("Released lock file")
 	return nil
 }
+
+// MigrateKeyValueStore attempts to idempotently migrate a kvstore at the
+// legacy location in to the new location as a one-time operation. If a
+// migration is not necessary, returns nil. If a migration is required but
+// does not succeed for any reason, returns an error.
+func (plugin *netPlugin) MigrateKeyValueStore() error {
```

Comment:
>MigrateKeyValueStore [](http://example.com/codeflow?start=25&length=20)

Can you add the test for this API? Feel free to add a work item to add it later if this requires more changes